docs/symptom-survey/survey-files.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Response Files
+parent: COVID Symptom Survey
+nav_order: 1
+---
+
+# Response Files
+{: .no_toc}
+
+Users with access to the [COVID symptom survey](./index.md) aggregate data
+should have received SFTP credentials for a private server where the data are
+stored. This documentation describes the survey data available on that user.
+
+You should have signed a Data Use Agreement with Facebook and with CMU to gain
+access to these files. If you have not, aggregate data is available
+[through the COVIDcast API](../api/covidcast-signals/fb-survey.md).
+
+Important updates for data users, including corrections to data or updates on
+data processing delays, are posted as `OUTAGES.txt` in the SFTP server directory
+where the data is accessible.
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Naming Conventions
+
+All dates in filenames are of the form `YYYY_mm_dd`.
+
+Cumulative files:
+
+	cvid_responses_{from}_-_{to}.csv.gz
```

Comment:
```suggestion
	`cvid_responses_{from}_-_{to}.csv.gz`
```

Reply:
Not needed in Markdown; indentation already turns this into a code block